src/sentry/api/endpoints/data_export.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import absolute_import
+
+from rest_framework.response import Response
+
+# from sentry import features
+from sentry.api.base import Endpoint
+from sentry.api.bases.incident import IncidentPermission
+from sentry.models import ExportedData
+
+
+class DataExportEndpoint(Endpoint):
+    permission_classes = (IncidentPermission,)
+
+    def post(self, organization, *args, **kwargs):
```

Comment:
Ok, so I understand what's going wrong now. A request is getting passed to `organization`, and the actual org isn't passed at all. To fix this, this endpoint needs to inherit from `OrganizationEndpoint`. You can do a search to see other uses of it.

Reply:
Ahhh okay, sorry for all the silly mistakes im still learning a lot about the codebase! You're a great help though, i will definitely make these changes